samples/ModulesSample/App.xaml.cs

```diff
@@ -33,7 +33,7 @@ public static AppBuilder BuildAvaloniaApp()
                 })
                 .With(new Win32PlatformOptions
                 {
-                    EnableMultitouch = true,
+                    //// EnableMultitouch = true,  // Not supported in Avalonia v11.0.0-preview4
```

Comment:
It is always enabled now

Reply:
Thank you @maxkatz6 for the heads up. I updated the code comment to reflect your note